src/client/datascience/notebook/notebookEditor.ts

```diff
@@ -98,8 +97,14 @@ export class NotebookEditor implements INotebookEditor {
         return getNotebookMetadata(this.document);
     }
     onExecutedCode?: Event<string> | undefined;
-    public getContent() {
-        return this.serializer.serializeNotebookDocument(this.document);
+    public async getContent(): Promise<string> {
+        const cells = generateCellsFromNotebookDocument(this.document, false);
```

Comment:
I don't think this function is the right one.  We should be using the one we had in the serializer, this function doesn't support converting Notebook cells to ipynb cells with the output. **The output is ignored**.